.gitlab-ci.yml

```diff
@@ -59,16 +59,24 @@ build-linux:
     TARBALL_EXT: tar.xz
     ADD_CABAL_ARGS: "--enable-split-sections"
 
-build-x86_64-linux-alpine:
+build-linux-alpine:
   extends: .build
+  parallel:
+    matrix:
+      - ARCH: i386
+        OS: [alpine3_12]
```

Comment:
no alpine3_15 then?

Reply:
Sadly not with the pinned version of the ci-images repo that is currently being used.

I think upgrading the pin would be wise, but perhaps not at this point in time?

Reply:
To be clear, it would be an easy thing to do, and I could even surgically only change the pin for the alpine builds.

Reply:
not that I can think of, let's keep using 3_12 :)